src/Microsoft.DocAsCode.MarkdigEngine.Extensions/CodeSnippet/HtmlCodeSnippetRenderer.cs

```diff
@@ -17,6 +17,10 @@ public class HtmlCodeSnippetRenderer : HtmlObjectRenderer<CodeSnippet>
     {
         private readonly MarkdownContext _context;
         private const string tagPrefix = "snippet";
+        private const string warningMessageId = "codeIncludeNotFound";
+        private const string defaultWarningMessage = "It looks like the sample you are looking for has moved!  Rest assured we are working on resolving this.";
```

Comment:
Are you going to modify template to include these messages?

Reply:
Yes. My change have checked in and localized: https://github.com/Microsoft/templates.docs.msft.zh-cn/blob/master/ContentTemplate/token.json#L75